docs/concepts/vector.rst

```diff
@@ -0,0 +1,26 @@
+Vector
+##################################
+
+Vector is lightweight and ultra-fast tool for building observability pipelines.
+In the aspects project it's used as a Ralph replacement, but can be used as
+a general purpose log collector.
+
+
+Vector Components
+-----------------
+
+Vector consists of the following components:
+
+- `Sources <https://vector.dev/docs/reference/sources/>`_ - Collects data from a source and sends it to Vector.
+- `Transforms <https://vector.dev/docs/reference/transforms/>`_ - Modifies events as they pass through Vector.
+- `Sinks <https://vector.dev/docs/reference/sinks/>`_ - Sends events to a destination.
+
```

Comment:
Can we add some details here about our implementation? Like:

Aspects has Sources configured for xAPI logging statements that are optionally generated by `event-routing-backends`, and for tracking log statements. Each of these have their own Transforms, which validate that the log statements are JSON and forward them to Sinks which store them in ClickHouse tables.